php-classes/Slate/CBL/Tasks/Task.php

```diff
@@ -2,11 +2,16 @@
 
 namespace Slate\CBL\Tasks;
 
+use DB;
+use HandleBehavior;
+use Slate\CBL\Skill;
+use Slate\CBL\Tasks\Attachments\AbstractTaskAttachment;
+
 class Task extends \VersionedRecord
 {
     //VersionedRecord configuration
     public static $historyTable = 'history_cbl_tasks';
-
+    
```

Comment:
@nbey watch the trailing whitespace, these are my "user settings" for VSCode that includes an option that prevents any trailing space from being saved:

```json
// Place your settings in this file to overwrite the default settings
{
	"editor.insertSpaces": true,
    "editor.renderWhitespace": true,
    "explorer.workingFiles.maxVisible": 20,
    "files.trimTrailingWhitespace": true
}
```